src/ws/cockpitauth.c

```diff
@@ -503,10 +503,25 @@ build_gssapi_output_header (GHashTable *headers,
 }
 
 static const gchar *
-auth_parse_application (const gchar *path,
-                        gchar **memory)
+application_parse_host (const gchar *application)
+{
+  const gchar *prefix = "cockpit+=";
```

Comment:
Why was there a need to introduce an additional '=' concept in the application name? I understand the need for prefixing this in the pretty URLs ... but why in this application suffix?